pkg/datagatherer/k8s/generic.go

```diff
@@ -110,6 +111,12 @@ func (g *DataGatherer) Fetch() (interface{}, error) {
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	// Redact Secret data
+	if strings.ToLower(g.groupVersionResource.Resource) == "secret" {
+		for i := range list.Items {
+			list.Items[i].Object["data"] = map[string]interface{}{}
```

Comment:
Also is it adequate to just work with the unstructured objects, or should they be properly converted to `Secrets`?

Reply:
Unstructured is fine, assuming the question/thread above is resolved IMO